Planning an offsite in just 8 weeks (#3952)
PostHog/posthog.com: contents/blog/planning-a-company-offsite.md

rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
featuredImage: ../images/blog/product-people.png
featuredImageType: full
author: ["grace-mckenzie"]
categories: ["Inside PostHog", "Guides"]
---

As a company born at the beginning of the pandemic, PostHog has been fully-remote from its inception. Our team is 30+ people distributed across 10 countries; we have no central office, but [we still value getting together](/blog/asynchronous-remote-companies) in person. 

We give employees [social and co-working budgets](/handbook/people/benefits) to encourage meetups and in-person onboarding. But the main way we facilitate in-person collaboration is [through offsites](/handbook/company/offsites) – small team offsites for focused work, and one all-company offsite per year for high-level strategic planning and cross-team interaction.  

In large companies, offsite planning is centralized within the People or Operations teams, but this isn’t possible in many early-stage startups. This guide is all about how to plan a killer offsite in just eight weeks – including a few tips and tricks I’ve learned over a decade of organizing team retreats.

> > Grace McKenzie is Operations Manager at PostHog. Add more copy here

Author: @postgrace Would you like to write a short bio here? A couple of sentences would be ideal and we can link to your PH profile / LinkedIn etc.

Reviewer: How's something like this?

"Grace McKenzie is the Operations Manager at PostHog, where she acts as a swiss-army-knife of sorts, tackling any and all back office projects that help build the scaffolding we need as our company scales. In her spare time, you can find her on the rugby pitch playing flanker or at her local billiards hall practicing her 8-ball breaks.

> > Grace McKenzie is Operations Manager at PostHog. Add more copy here

Author: ```suggestion
> Grace McKenzie is the Operations Manager at PostHog, where she tackles any and all back office projects that help build the scaffolding we need as our company scales. In her spare time, you can find her on the rugby pitch playing flanker or at her local billiards hall practicing her 8-ball breaks.
```

---

> date: 2022-08-16

Author: ```suggestion
date: 2022-08-17
```

Here’s [one we made earlier](/handbook/company/offsites#how-to-plan-an-offsite-in-8-weeks---a-checklist) – feel free to use it.

Seriously, a comprehensive checklist is a must for offsite planning. We made this list to help team leaders organize their small team offsites independently. Our three person “Plops” (that’s People & Operations) team help where possible, but with nine small teams all organizing one or more meetups a year, we need people to take ownership.

The checklist is structured as an eight-week timeline with items to check off at eight weeks, seven weeks and so on. It also includes post-offsite items for collecting feedback – we’ve learnt a lot about running offsites from gathering feedback on them!

Here are some key items to highlight:

- Centralizing flight booking is a huge headache, so don’t do it. We give all team members company cards for this reason – see our [startup ops kit](/blog/startup-ops-toolkit) for more on this. This also means you don’t have to gather up lots of sensitive info (passport numbers etc.).  

> - Decide in advance who is going to record the notes from offsites sessions. It is really easy for post-offsite followup to fall through the cracks, which means a bunch of time and effort wasted. 

Reviewer 2: ```suggestion
- Decide in advance who is going to record the notes from offsites sessions. It is really easy for post-offsite follow-up to fall through the cracks, which means a bunch of wasted time and effort. 
```

- It’s good if the offsite organizer arrives a day early to prep for the team's arrival. Whether it’s grocery shopping or troubleshooting airport transfers, it just makes everyone’s experience smoother.

## Choosing a location

<div style="width:100%;height:0;padding-bottom:75%;position:relative;"><iframe src="https://giphy.com/embed/1jv9YQIX4cvtRAHVRy" width="100%" height="100%" style="position:absolute" frameBorder="0" class="giphy-embed" allowFullScreen></iframe></div>

> 

Reviewer 2: ```suggestion
```

- **Affordability:** An ideal offsite location depends on the stage of the company – a realistic budget for a five-person company won’t be the same as one for 100. Work out how much you can spend per person on travel and accommodation for your chosen location as soon as possible. And don’t go to Switzerland – it will bankrupt you.

<NewsletterForm compact />

## Planning your schedule

<div style="width:100%;height:0;padding-bottom:194%;position:relative;"><iframe src="https://giphy.com/embed/elDpvxPF5FfcBbwOvR" width="100%" height="100%" style="position:absolute" frameBorder="0" class="giphy-embed" allowFullScreen></iframe></div>

It’s tempting to maximize your time together by packing the itinerary full of brainstorming, planning, and breakout sessions. Don’t overdo this. In reality, having a good balance between sessions, free social time, and fun activities is important to avoid burning your team out. 

> In my experience, planning for a majority of unstructured time can lead to really amazing brainstorming and ideation. People will inevitably discuss their work as it’s a major thing that everyone has in common.

Reviewer 2: ```suggestion
In my experience, planning for a majority of unstructured time can lead to really amazing brainstorming and ideation. People will inevitably discuss their work anyway as it’s something that everyone has in common.
```

One mainstay event that we do at every all-company offsite is a 24-hour hackathon, where our team comes up with the most outlandish ideas possible, and self-organizes into interdisciplinary teams to produce an MVP by the following day. Some core features of our product, like the ability to view your users on a world map; and some seriously silly things, like a children’s book focused on product analytics, have been born from this highly-rated session.

> > **Pro Tip:** Encourage people to take time-off either before or after the offsite and use their company-sponsored travel for a lower-cost vacation in an interesting locale. After our Iceland offsite, my teammate Neil and I rented a camper van and traveled around the [iconic Ring Road](https://www.nordicvisitor.com/blog/5-things-to-know-about-icelands-ring-road/) for a week! Other team members arranged their own excursions together. It’s a great way to create social connections.

Author: ```suggestion
> **Pro Tip:** Encourage people to use their company-sponsored travel for a lower-cost vacation in an interesting locale. After our Iceland offsite, my teammate Neil and I rented a camper van and traveled around the [iconic Ring Road](https://www.nordicvisitor.com/blog/5-things-to-know-about-icelands-ring-road/) for a week! Other team members arranged their own excursions together. It’s a great way to create social connections.
```

Budgeting doesn’t need to be the first thing you consider, but it should definitely be on the earlier side. I recommend coming up with a draft budget about a month and a half out, and finalizing it after a gut check with your manager or another member of your team by the end of the following week.

Numbers can vary wildly depending on location, accommodations, activities, team size, and a multitude of other factors. In my experience, the following benchmarks can be useful in initial projections before you’ve finalized any of the above factors:

- Accommodations = $200/night/person
- Intercontinental airfare = $1,000/person
- Intracontinental airfare = $500/person
- Ground transportation = $50/day/person
- Food & drinks = $50/day/person
- Contingency = 10% of total budget

As a reference, we budget $1,500 per person for a small team offsite and $3,000 for an all-company offsite. Our last all-company offsite with 20 people in Reykjavik (one of the priciest cities in the world) came in at just under $30k, or approximately $2,800 per person.

## Final thoughts

> Offsite planning can often be some of the hardest, yet most rewarding work you can do in a startup. I hope the information above helps you along the way, but be sure to check out [our open-source guide](handbook/company/offsites#how-to-plan-an-offsite-in-8-weeks---a-checklist) for how we plan an offsites in eight weeks. It includes our comprehensive checklist and some public templates you can use to make the process as efficient as possible.

Reviewer 2: ```suggestion
Offsite planning can often be some of the hardest, yet most rewarding work you can do in a startup. I hope the information above helps you along the way, but be sure to check out [our open-source guide](handbook/company/offsites#how-to-plan-an-offsite-in-8-weeks---a-checklist) for planning offsites in eight weeks. It includes our comprehensive checklist and some public templates you can use to make the process as efficient as possible.
```
